src/webots/gui/WbSimulationView.cpp

```diff
@@ -746,24 +753,24 @@ void WbSimulationView::realTime() {
   WbSimulationState::instance()->setMode(WbSimulationState::REALTIME);
 }
 
-void WbSimulationView::run() {
-  WbSimulationState::instance()->setMode(WbSimulationState::RUN);
-}
-
 void WbSimulationView::fast() {
   WbSimulationState::instance()->setMode(WbSimulationState::FAST);
 }
 
-void WbSimulationView::disableFastMode(bool disabled) {
-  WbActionManager::instance()->action(WbAction::FAST)->setEnabled(!disabled);
-  mView3D->setUserInteractionDisabled(WbAction::DISABLE_FAST_MODE, disabled);
+void WbSimulationView::disableRendering(bool disabled) {
+  WbActionManager::instance()->action(WbAction::RENDERING)->setEnabled(!disabled);
```

Comment:
Do you think we should also turn off the rendering when the DISABLE_RENDERING option is checked?

Reply:
Previously, it was just locking the `FAST` state, but I agree that turning the rendering off makes more sense

Reply:
Implemented. I made more changes than expected as the rendering state has to be restored from the current session and from the `.wbproj` file. So let me know if it is good.

Reply:
Thank you! It works fine

Reply:
Also, the changelog was updated before (the last item):
https://github.com/cyberbotics/webots/blob/a76b3502fd24dd5b619174874ba3c9246f32d7ae/docs/reference/changelog-r2021.md

Reply:
Does it look good now, can we merge?